socorro/scripts/fetch_crash_data.py

```diff
@@ -69,55 +73,53 @@ def fetch_crash(outputdir, api_token, crash_id):
 
     raw_crash = resp.json()
 
-    # Fetch dumps
-    dumps = {}
-    dump_names = raw_crash.get('dump_checksums', {}).keys()
-    for dump_name in dump_names:
-        print('Fetching %s -> %s' % (crash_id, dump_name))
-        if dump_name == 'upload_file_minidump':
-            dump_name = 'dump'
-
-        resp = requests.get(
-            HOST + '/api/RawCrash/',
-            params={
-                'crash_id': crash_id,
-                'format': 'raw',
-                'name': dump_name
-            },
-            headers=headers,
-        )
-
-        if resp.status_code != 200:
-            raise Exception('Something unexpected happened. status_code %s, content %s' % (
-                resp.status_code, resp.content)
-            )
-
-        dumps[dump_name] = resp.content
-
-    # Save everything to file system in the right place
-    # http://antenna.readthedocs.io/en/latest/architecture.html#aws-s3-file-hierarchy
-
```

Comment:
This code didn't get changed at all--just moved so that we fetch and save the raw crash data and then fetch and save the dumps (if warranted).